_posts/2020-10-13-from-aligned-to-unaligned-checkpoints-part-1.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post 
+title: "From Aligned to Unaligned Checkpoints - Part 1: Checkpoints, Alignment, and Backpressure" 
+date: 2020-10-13T03:00:00.000Z
```

Comment:
```suggestion
date: 2020-10-15T03:00:00.000Z
```

Reply:
If not to be published today, please update accordingly when merging (_ideally_, also the file name and such).